ssh/session/session.go

```diff
@@ -27,7 +27,7 @@ var (
 	ErrFirewallBlock = fmt.Errorf("you connot connect to this device because a firewall rule block your connection")
 	ErrHost          = fmt.Errorf("failed to get the device address")
 	ErrFindDevice    = fmt.Errorf("failed to find the device")
-	ErrDial          = fmt.Errorf("failed to tunnel the server to agent")
+	ErrDial          = fmt.Errorf("failed to tunnel the server to agent. the device can be offline")
```

Comment:
Please use: failed to connect to device agent, please check the device connection

Reply:
Perfect

Reply:
https://github.com/shellhub-io/shellhub/pull/2161/files#diff-c4ddc485bc4aee5bf70bf7c489310bb48e38c6ea908529464013c17ef40df86dR30